### PR TITLE
[DEPENDABOT] Resolve update of SDK

### PR DESCRIPTION
**Pull request by VicThorX**
These changes allow us to fix this dependabot PR: #33

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -10,7 +10,7 @@ WORKDIR /src
 COPY ./*.sh ./
 RUN shellcheck -e SC1091,SC1090 ./*.sh
 
-FROM mcr.microsoft.com/dotnet/core/sdk:3.1.302-buster AS restore
+FROM mcr.microsoft.com/dotnet/core/sdk:3.1 AS restore
```
Comment (andresmoschini) [inline Dockerfile:13]: What do you think about keeping buster?

```suggestion
FROM mcr.microsoft.com/dotnet/core/sdk:3.1-buster AS restore
```

**Comment by andresmoschini** [comment]
Could you also change the base image version of the final image?

image

in my opinion, it could be `mcr.microsoft.com/dotnet/core/aspnet:3.1-buster-slim`

File: cuit-service.sln [hunk 2]
```diff
@@ -18,7 +18,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		build-n-publish.sh = build-n-publish.sh
 		demo.http = demo.http
 		Dockerfile = Dockerfile
-		global.json = global.json
```
Comment (andresmoschini) [inline cuit-service.sln:21]: If I am not wrong, deleting `global.json` file is missing.

Comment (VicThorX) [reply cuit-service.sln]: I need only need modify the content of the global.json, not delete it! haha

Comment (andresmoschini) [reply cuit-service.sln]: No, no, you need to delete it!

File: global.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "3.1.302"
+    "version": "3.1"
```
Comment (andresmoschini) [inline global.json:3]: Please delete global.json file.

**Pull request merged**